playbooks/tmp/example_device.json

```diff
@@ -1,18 +1,18 @@
 {
     "ANSIBLE_MODULE_ARGS": {
-        "device_name": "acc_geg_100",
-        "device_sample_rate": 10,
-        "device_snmp_community": "kentik",
-        "device_snmp_ip": "10.1.7.200",
-        "device_subtype": "router",
-        "minimize_snmp": false,
-        "plan_name": "Free Flowpak Plan 2",
-        "sending_ips": ["10.1.7.200"],
-        "site_name": "GEG",
-        "device_bgp_type": "none",
+        "deviceName": "access_switch_100",
+        "deviceSampleRate": 10,
+        "deviceSnmpCommunity": "kentik",
+        "deviceSnmpIp": "192.168.1.100",
```

Comment:
Probably should use IP addresses in the example that are from the RFC5737 space reserved for that purpose (https://datatracker.ietf.org/doc/rfc5737/).

Reply:
done

Reply:
also renamed the tmp directory to files to match ansible best practices. does that work?

Reply:
https://docs.ansible.com/ansible/latest/dev_guide/developing_collections_structure.html